Makefile

```diff
@@ -29,10 +28,11 @@ next_version:
 bridge_version:
 	# Set Kafka Bridge version to its own version
 	$(FIND) ./install -name '*.yaml' -type f -exec $(SED) -i '/value: "\?strimzi\/kafka-bridge:[a-zA-Z0-9_.-]\+"\?/s/strimzi\/kafka-bridge:[a-zA-Z0-9_.-]\+/strimzi\/kafka-bridge:$(BRIDGE_VERSION)/g' {} \;
-	# Update Kafka Bridge image tag in chart values.yaml to BRIDGE_VERSION
-	$(SED) -i '/name: kafka-bridge/{n;s/\(tag: \).*/\1$(BRIDGE_VERSION)/g}' $(CHART_PATH)values.yaml
-	# Update Kafka Bridge image tag in chart README.md config grid with BRIDGE_VERSION
-	$(SED) -i 's/\(kafkaBridge.image\.tag[^\n]*| \)`.*`/\1`$(BRIDGE_VERSION)`/g' $(CHART_PATH)README.md
+	for HELM_VERSION in 2 3	; do	\
+		CHART_PATH=./helm-charts/helm$$HELM_VERSION/strimzi-kafka-operator	;	\
+		$(SED) -i '/name: kafka-bridge/{n;s/\(tag: \).*/\1$(BRIDGE_VERSION)/g}' $$CHART_PATH/values.yaml	;	\
+		$(SED) -i 's/\(kafkaBridge.image\.tag[^\n]*| \)`.*`/\1`$(BRIDGE_VERSION)`/g' $$CHART_PATH/README.md	;	\
+	done
```

Comment:
What's with all the whitespace around `;`  and before `\`? This comment applies to the other change in this file too, of course.

Reply:
Fixed.